src/components/Box/Box.tsx

```diff
@@ -24,13 +24,13 @@ interface BoxProps
     extends SpaceProps,
         LayoutProps,
         PositionProps,
-        ColorProps,
+        Omit<ColorProps, 'color'>, // HACK: avoid collision of `color` prop
         FlexboxProps,
         GridProps,
         BackgroundProps,
         TextAlignProps {}
 
-const Box: StyledComponentBase<'div', never, BoxProps, 'theme'> = styled.div.attrs({ theme })<BoxProps>`
+const Box: StyledComponentBase<'div', DefaultTheme, BoxProps, 'theme'> = styled.div.attrs({ theme })<BoxProps>`
```

Comment:
Have you tried stripping off the whole `StyledComponentBase` type? I think Typescript can derive the correct type from the assignment

Reply:
Then the project won't compile, because of other component that depend on `Box` breaking. I would like to have it this way, though. Looking into it now.

Reply:
This is done now in c525f49 and 0405c22